docs/docs/guides/migrate/sources/auth0-guide.md

```diff
@@ -0,0 +1,188 @@
+
+To call the ZITADEL Admin API, you need to authenticate using a **Service User** with the `IAM_OWNER` Manager permissions.
+
+There are two recommended authentication methods:
+
+- **Client Credentials Flow**  
+  Learn how to authenticate with client credentials.
+
+- **Personal Access Token (PAT)**  
+  Learn how to create and use a PAT.
+
+**Reference:** [Service Users & API Authentication](https://docs.zitadel.com/)
+
+---
+
+### 5.2. Import Data with the ZITADEL API
+
+Use your **access token** or **PAT** to authenticate.
+
+Call the **Admin API – Import Data** endpoint, passing your generated JSON file.
+
+Verify that the users were imported successfully in the ZITADEL console.
+
+**Import Endpoint:**
+
+- `POST /admin/v1/import`
+- `Authorization: Bearer <token>`
+- **Body:** Generated in step 4.2
+
+#### Example cURL request
+
+```bash
+curl --location 'https://<instance-domain>/admin/v1/import' \
+--header 'Content-Type: application/json' \
+--header 'Accept: application/json' \
+--header 'Authorization: Bearer <access-token>' \
+--data-raw '{
+  "dataOrgs": {
+    "orgs": [
+      {
+        "orgId": "<your-org-id>",
+        "humanUsers": [
+          {
+            "userId": "auth0|emxdpVxozXeFb1HeEn5ThAK8",
+            "user": {
+              "userName": "tommie_krajcik85@hotmail.com",
+              "profile": {
+                "firstName": "Tommie Krajcik",
+                "lastName": "Tommie Krajcik"
+              },
+              "email": {
+                "email": "tommie_krajcik85@hotmail.com",
+                "isEmailVerified": true
+              },
+              "hashedPassword": {
+                "value": "$2b$10$d.GvZhGwTllA7OdAmsA75uGGzqr/mhdQoU88M3zD.fX3Vb8Rcf33."
+              }
+            }
+          }
+        ]
+      }
+    ]
+  },
+  "timeout": "30m0s"
+}'
+```
+
+## 6. Testing the Migration
+
+### 6.1. Test User Login
+
+Use the **ZITADEL login page** or your integrated app to test logging in with one of the imported users.
+
+> **Password for the sample user:** `Password1!`
+
+Confirm that the migrated password works as expected.
+
+---
+
+### 6.2. Troubleshooting
+
+**Common issues:**
+
+- Missing password hashes  
+- Malformed JSON  
+- Invalid or incomplete user data  
+
+The import endpoint returns an `errors` array which can help you identify any issues with the import.
+
+#### Where to check logs and get help
+
+You can also verify that a user was imported by calling the **events endpoint** and checking for the following event type:
+
+```json
+"user.human.added"
+```
+
+## 7. Q&A and Further Resources
+
+### Real-World Scenarios & Common Questions
+
+**Q:** What is the maximum number of users that can be imported in a single batch?  
+**A:** There is no hard limit on the number of users. However, there is a **timeout**.  
+For **ZITADEL Cloud deployments**, the timeout is **5 minutes**, which typically allows for importing around **5,000 users per batch**.
```

Comment:
This can be confusing since the timeout in the http call is set to 30mins.

Can we avoid confusion here?